btclightclient/btc_light_client_test.go

```diff
@@ -95,6 +102,51 @@ func TestInsertHeader(t *testing.T) {
 	}
 }
 
+func TestLatestFinalizedBlock(t *testing.T) {
+	commonTestCase := CommonTestCases()
+	testCases := map[string]FinalizedTestCaseData{
+		"Append a fork":                      {nil, 11, "6393fcb4ba7189f914c2f74fad2bd0ef7743c867b4a81db458f9f9e506458fe7"},
+		"Create fork":                        {nil, 10, "7a47c3a083add37394061eba8dbfb1fe2026d3fed6bd3d428b043b515bcb269e"},
+		"Insert failed because fork too old": {ErrForkTooOld, 10, "7a47c3a083add37394061eba8dbfb1fe2026d3fed6bd3d428b043b515bcb269e"},
+		"Block doesn't belong to any fork!":  {ErrParentBlockNotInChain, 10, "7a47c3a083add37394061eba8dbfb1fe2026d3fed6bd3d428b043b515bcb269e"},
```

Comment:
please explain this hashes. From reading this test, I don't know if the test case is true or not.